Controller/FormController.php

```diff
@@ -174,6 +175,50 @@ public function postAction(Request $request): Response
         return $this->handleView($this->view($this->getApiEntity($entity, $locale), 201));
     }
 
+    public function postTriggerAction(Request $request, int $id): Response
+    {
+        $action = $request->query->get('action');
+        $locale = $this->getLocale($request);
+        $entity = $this->formManager->findById($id, $locale);
+
+        if (!$entity) {
+            throw new NotFoundHttpException(sprintf('No form with id "%s" was found!', $id));
+        }
+
+        try {
+            switch ($action) {
+                case 'copy':
+                    $copiedForm = null;
+                    foreach ($entity->getTranslations() as $translation) {
```

Comment:
i implemented this here because the existing `getApiEntity` method made it really easy to do it 😇 

should we create a `copy` method on the `FormManager` instead? if yes, we need to move the `getApiEntity` method to the `FormManager` too or we cannot easily reuse the existing save method

Reply:
I moved it to the manager. also it did make problems when using save when the form has multiple languages.